### ISSUE TITLE
Reload Config Triggers "Not Running" Popup

### ISSUE THREAD

**Issue by henworth** [toni-moreno/snmpcollector#127]
When clicking "Reload Config" a popup displays with the message "Server seems not being running..." After clicking Ok a few times, another shows up with 
```
{
  "isTrusted": true
}
```

After clicking Ok there, either the same thing will happen again or control of the page will be possible again. However, the Runtime page is empty and the only way to get it back is to restart the daemon.

Only logs (with debug enabled) look to be:

```
snmpcollector[8472]: time="2017-02-08 11:00:15" level=info msg="trying to reload configuration for all devices"
snmpcollector[8472]: time="2017-02-08 11:00:15" level=info msg="RELOADCONF: begin device Gather processes stop..."
```

**Comment by toni-moreno** [comment]
Can you give us more accurate information on this error? .

can you show us your main config.toml config file? 
how many devices have you configured? can you show us your runtime view?
thanx a lot

**Comment by henworth** [comment]
I haven't changed anything but the logLevel in the config file, it's otherwise the defaults. Only configured three devices, two Cisco MDS switches and one EMC DataDomain. The latter is currently disabled as it seems to cause the daemon to die and be restarted by systemd.

How much detail from the runtime view do you need?

**Comment by toni-moreno** [comment]
Can you configure device log level to debug and send us all generated log files in the log directory ? ( remove all sensitive data before if you wish )

There is any other related log in the system log /var/log/messages file ?

**Comment by toni-moreno** [comment]
Hi @henworth any news about the crash?

**Comment by sbengo** [comment]
Hi @henworth,

We have added web UI block to avoid click more than time the reload_config button until the the first one has already finished. It will be implemented on the next release (0.6.5).

Please, if you have some news or info to keep studying the case, attach them or this issue will be closed.

Thanks,
Regards!

**Comment by toni-moreno** [comment]
Hi @henworth I  will close this issue. Feel free to reopen if you have again the same crash.
